content/blog/getting-started-with-hpe-greenlake-api-for-virtualization.md

```diff
@@ -0,0 +1,411 @@
+    "resourceUri": "/data-services/v1beta1/async-operations/9825d5e8-5213-433b-8fbf-1945bb4496e1",
+    "rootTask": {
+        "id": "9825d5e8-5213-433b-8fbf-1945bb4496e1",
+        "name": "",
+        "resourceUri": "/data-services/v1beta1/async-operations/9825d5e8-5213-433b-8fbf-1945bb4496e1",
+        "type": "task"
+    },
+    "services": [
+        "private-cloud-business-edition"
+    ],
+    "sourceResource": {
+        "name": "VVOL-ds2",
+        "resourceUri": "/api/v1/datastores/fdcc724f-3f22-5fa6-8477-81f0fd701512",
+        "type": "Datastore"
+    },
+    "startedAt": "2024-03-29T00:56:16.981298562Z",
+    "state": "SUCCEEDED",
+    "subtreeTaskCount": 2,
+    "suggestedPollingIntervalSeconds": 30,
+    "type": "task",
+    "updatedAt": "2024-03-29T00:59:23.013384523Z",
+    "userId": "ronald.dharma@hpe.com"
+}
+```
+
+*T﻿he above figure shows the termination of machine-instance in AWS reached 100% progress and completed successfully.*
+
+## Summary
+
+This blog post introduces you to the new set of HPE Greenlake API for data services named virtualization to support resources from a hypervisor that includes VMware (6.7.X, 7.X and 8.X) for on-premises, AWS EC2 and Azure for cloud-public-provider. This API set will evolve throughout next release in the future. The March 2024 announcement introduces V1Beta1 of the API version. This set of APIs is documented at HPE GreenLake Developer [website ](https://developer.greenlake.hpe.com) and it is available for download as Open API 3.1.X specification in JSON file.
+
+In this post, I started by introducing the HPE GreenLake virtualization API on the API documentation page. Afterward, I provided a couple of tips on using this Virtualization REST API such as discovery of the on-premises resources and determined the services that were applied against the VMware cluster. In the following, I provided a complete guides to deploy a Virtual Machine inside the Cloud Service Provider (AWS). Lastly, I also introduced an example on how to invoke DEL method of the Virtual Machines at the Cloud to clean up the deployed Virtual Machines on the Cloud Service Provider. As a bonus, I also provided example to deploy datastores in the on-premises VMware infrastructure part of the HPE GreenLake Private Cloud Business Edition. In future blogs, I will provide examples that incorporate HPE GreenLake APIs for virtualization with different sets of APIs from different data services on HPE GreenLake. 
+
+Please don’t hesitate to explore this new set of HPE GreenLake API for virtualization and see how you can improve your agility in managing your data. Any questions on HPE GreenLake API for virtualization? Or do you have any suggestions or cool ideas that you want share with the community? Please [join](https://developer.hpe.com/slack-signup/) the HPE Developer Slack Workspace and start a discussion in our #hpe-greenlak-data-services [slack channel](https://hpedev.slack.com/archives/C02D6H623JP).
```

Comment:
Virtualization or virtualization? For consistency throughout your post I would go with a capital "V" for "HPE GreenLake API for Virtualization".

Please don’t hesitate to explore this new set of HPE GreenLake API for Virtualization and see how you can improve your agility in managing your data. Any questions on HPE GreenLake API for Virtualization?